app/views.py

```diff
@@ -279,19 +279,19 @@ def eval_card(request, card_name):
     except ValueError as e:
         return HttpResponse(json.dumps({
             'error': e.message
-        }), mimetype="application/json")
+        }), content_type="application/json")
     except DeadlineExceededError:
         return HttpResponse(json.dumps({
             'error': 'Computation timed out.'
-        }), mimetype="application/json")
+        }), content_type="application/json")
```

Comment:
I think its better to use JsonResponse.

Reply:
Beyond the scope, [See this](https://github.com/sympy/sympy_gamma/pull/135#issuecomment-612584314)